libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/InstanceView/InstanceView.tsx

```diff
@@ -357,4 +367,52 @@ export class InstanceView extends React.Component<
     };
     this.setState(reloadDataFunc);
   }
+
+  private onRenderLabel = (type: SelectionType) => (
+    p: IChoiceGroupOption | undefined
+  ) => {
+    const classNames = InstanceViewStyles();
+    let selectionText = "";
+    switch (type) {
+      case SelectionType.AllSelectionType:
+        selectionText =
+          this.state.selectionDetails.selectedAllSelectedIndexes.length +
+          " selected";
+        break;
+      case SelectionType.CorrectSelectionType:
+        selectionText =
+          this.state.selectionDetails.selectedCorrectDatasetIndexes.length +
+          " selected";
+        break;
+      case SelectionType.IncorrectSelectionType:
+        selectionText =
+          this.state.selectionDetails.selectedIncorrectDatasetIndexes.length +
+          " selected";
+        break;
+      default:
+        break;
+    }
+    const stackItemStyles: IStackItemStyles = {
+      root: classNames.stackItemsStyle
+    };
+    const textStyle = {
+      root: classNames.selectedTextStyle
+    };
+    return (
+      <div style={{ display: "flex", flexDirection: "column" }}>
```

Comment:
this can be replaced by a horizontal stack

Reply:
seems not needed at all though.

Reply:
removed